helm-maintainers-onboarding-guide.md

```diff
@@ -61,7 +61,7 @@ Code:
 - Protobuf files MUST NOT have mandatory fields added
 - The public API of anything in `pkg/` SHOULD NOT be modified
 - Go interfaces in `pkg/` MUST NOT be modified
-- The first release of a major feature addition SHOULD be hidden behind an `experimental` feature flag (see Tiller rudders)
```

Comment:
Even though Tiller was removed, I still think the idea of pointing at an example experimental feature flag is useful. Can we replace this with something else, like the OCI experimental flag?